data/sql_updates/post_totals_create_candidate_aggregates.sql

```diff
@@ -8,7 +8,8 @@ with totals as (
         receipts,
         disbursements,
         last_cash_on_hand_end_period,
-        last_debts_owed_by_committee
+        last_debts_owed_by_committee,
+        cast('false' as boolean) as federal_funds_flag
```

Comment:
At one point I was getting a casting error and I think I was over-zealous about getting rid of it. I can simplify that.